ansible_collections/arista/avd/extensions/molecule/example-isis-ldp-ipvpn/converge.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Converge
```

Comment:
hmmm so we need to have the intermediate playbook?

I think you can do this accordong to doc cf next comment